src/modules/assessments/components/household/HouseholdSummaryTabPanel.tsx

```diff
@@ -94,10 +94,9 @@ const HouseholdSummaryTabPanel = memo(
           refetch();
 
           tabs.forEach(({ clientId, enrollmentId }) => {
-            // Make it so the "Assessments" table on the Enrollment page gets refetched next time
+            // Fully refetch the Enrollment
             cache.evict({
               id: `Enrollment:${enrollmentId}`,
```

Comment:
@gigxz you were right that the real problem here was that `enrollment.intakeAssessment` was still `null`.

I'm not sure if there really was an issue with `householdSize` or I imagined that/was mistaken. But I removed the addition of `householdSize` that you recommended [here](https://github.com/greenriver/hmis-frontend/pull/745/commits/e00955818faa5b9a44ee93f15594fb1eb999fc21#diff-47969bb3d13a8aabb25165358ccddd0702bba766506a95eadf9c875dd5899507R34), because of build fails that it introduced in other files (see [here](https://github.com/greenriver/hmis-frontend/pull/745/commits/12fa56fb9d5e3532dedf709bd8c15d6a9fd4ba9d)).

With this change, when I tried to reproduce the issue and checked in the debugger for the value of `enrollment.householdSize` [here](https://github.com/greenriver/hmis-frontend/pull/745/commits/e00955818faa5b9a44ee93f15594fb1eb999fc21#diff-0572cdaae28be2c9fec71c4b73468d9810eaa7bfaa99396a13f3e3186f7aeba8R29), it looks like it correctly updates from 2 to 1.

Reply:
Alright interesting, thanks for tracking that down!